doc/sphinx/README.rst

```diff
@@ -519,7 +517,7 @@ The ``.. coqtop::`` directive does *not* reset Coq after running its contents.
 
       Lemma l2: 2 + 2 <> 1.
 
-Add either ``undo`` to the first block or ``reset`` to the second block to avoid nesting lemmas.
+Add ``reset`` to the second block to avoid nesting lemmas.
```

Comment:
I suggest providing instead the advice of inserting a block like this:

```rst
.. coqtop:: none

   Abort.
```

In fact, we should even provide an `abort` option that would run an `Abort All.` automatically at the end of the block (but that's future work).